src/commands/app/db/collection/deleteOne.js

```diff
@@ -19,27 +19,19 @@ export class DeleteOne extends DBBaseCommand {
     const { collection, filter } = this.args
 
     try {
-      // Parse filter JSON
-      let filterObj
-      try {
-        filterObj = JSON.parse(filter)
-      } catch (error) {
-        this.error(`Invalid filter JSON: ${error.message}`)
-      }
-
       this.log(chalk.blue(`Deleting document from collection '${collection}'...`))
 
       const client = await this.db.connect()
       const coll = client.collection(collection)
 
       // Delete the document
-      const result = await coll.deleteOne(filterObj)
+      const result = await coll.deleteOne(filter)
 
       this.debugLogger?.info?.('Document deleted successfully:', result)
 
       const response = {
         collection,
-        filter: filterObj,
+        filter: filter,
```

Comment:
`filter: filter` fails the project's linting.

```suggestion
        filter,
```